Spigot-Server-Patches/0002-Paper-config-files.patch

```diff
@@ -81,7 +105,7 @@ index 0000000000000000000000000000000000000000..b8868b86338ce0e89bc74eccccf714b9
 +            while (iterator.hasNext()) {
 +                String s1 = (String) iterator.next();
 +
-+                if (matches(last, s1)) {
++                if (matches(last, s1) && sender.hasPermission(BASE_PERM + s1) || sender.hasPermission("bukkit.command.paper")) {
```

Comment:
`false && false || true` evaluates to `true` because `&&`s associativity is higher; this is equivalent to `(false && false) || true`, and the subcommand will always be included if the player has `bukkit.command.paper`, regardless of match.

Reply:
ah yes you are right. Will fix